src/zope/meta/config_package.py

```diff
@@ -348,7 +371,7 @@ def pre_commit_config_yaml(self):
             "pre-commit-config.yaml.j2",
             self.path / ".pre-commit-config.yaml",
             self.config_type,
-            oldest_python_version=OLDEST_PYTHON_VERSION.replace(".", ""),
+            oldest_python_version=self.oldest_python.replace(".", ""),
```

Comment:
Will this break should we ever get a version component number >= 10?

Reply:
I don't understand this comment. This line removes dots. It will work the same with a version number `10.5` as it does with `3.8`. Or am I misunderstanding something?